docs/4.0/docs/design/imagehub.md

```diff
@@ -0,0 +1,74 @@
+# imagehub
```

Comment:
Docs file need to be english version, PLS add eng version here, and put this markdown content to `i18n/zh-Hans/design/imagehub.md`

Reply:
ok